src/lib/resource/sections/applicant-select-reasons-for-the-delay-in-making-your-application.js

```diff
@@ -51,6 +53,10 @@ module.exports = {
                                 'Briefly explain reasons for the delay in making your application'
                         }
                     }
+                },
+                'help-reason-for-delay': {
+                    description:
+                        '{% from "components/details/macro.njk" import govukDetails %}{{ govukDetails({summaryText: "Help with why we ask about the application not being sent until now",html: \'<h2 class="govuk-heading-s">If the victim was an adult at the time of the crime (18 years or older)</h2><p class="govuk-body">The application must be sent as soon as is reasonably practicable and in any event, no later than 2 years after the crime.</p><h2 class="govuk-heading-s">If the victim was a child at the time of the crime (under 18 years old)</h2><p class="govuk-body">The application must be received:<ul class="govuk-list govuk-list--bullet"><li>On or before their 20th birthday, if the crime was reported to police before their 18th birthday</li><li>within 2 years of the crime first being reported to police, if the incident was reported on or after their 18th birthday</li></ul></p><h2 class="govuk-heading-s">Extending the time limits</h2><p class="govuk-body">We can only extend the time limits where:<ul class="govuk-list govuk-list--bullet"><li>due to exceptional circumstances, the application could not be made earlier (for example, where injuries stopped the victim applying)</li><li>the evidence provided to support the application means it can be decided without further extensive enquiries</li></ul></p><p class="govuk-body">You may have to provide evidence to support any reason for a delay in applying. This evidence may include a letter from your GP or other medical professionals.</p><p class="govuk-body">Any evidence to support your application should:<ul class="govuk-list govuk-list--bullet"><li>show why this application could not have been made earlier and explain the entire period of the delay  </li><li>provide us with enough information to make a decision on an application without the need for further extensive enquiries</li></ul></p>\'})}}'
```

Comment:
@neil-stephen-mcgonigle 3 changes:

1. 1st bullet - "On" should be all lower case
2. Bullets under "Extending the time limit" - "and" to be added between the bullets.
3. Paragraph under "Extending the time limit" - remove "s" from professionals
I'll get Graham to update the prototype to reflect these. 
Thanks